src/market/directory.rs

```diff
@@ -44,8 +87,8 @@ pub async fn sync_maker_addresses_from_directory_servers(
     network: Network,
 ) -> Result<Vec<MakerAddress>, DirectoryServerError> {
     // https://github.com/seanmonstar/reqwest/blob/master/examples/tor_socks.rs
-    let proxy =
-        reqwest::Proxy::all(format!("socks5h://{}", TOR_ADDR)).expect("tor proxy should be there");
+    let proxy = reqwest::Proxy::all(format!("socks5h://{}", TOR_SOCKS_ADDR))
+        .expect("tor proxy should be there");
```

Comment:
It seems this entire function is redundant now. We have better syncing code in taker API.